R/sandbox.R

```diff
@@ -0,0 +1,72 @@
+# library(magrittr)
```

Comment:
sandbox should be in inst and not included in the package if your going to have it. only functions in `R`

Reply:
ooh woops. I didn't even want to commit that at all.... thanks

Reply:
ah no worries. might be worth adding *sandbox.R to the gitignore